tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -112,6 +112,7 @@ def tap_comment(self):
         element.tap()
         self.marionette.execute_script(
             'arguments[0].scrollIntoView(true);', [element])
+        Wait(self.marionette, timeout=120).until(expected.element_displayed(*self._comment_locator))
         element = self.marionette.find_element(*self._comment_locator)
```

Comment:
You could do this here:
+        Wait(self.marionette, timeout=120).until(expected.element_displayed(element))
That is more efficient.